custom/rch/models.py

```diff
@@ -0,0 +1,212 @@
+import datetime
+from django.db import models
+from custom.rch.utils import fetch_mother_beneficiaries_records, fetch_child_beneficiaries_records
+
+
+STATE_DISTRICT_MAPPING = {
+    '28': [
+        '523'
+    ]
+}
+
+
+class RCHRecord(models.Model):
+    Caste = models.CharField(null=True, max_length=255)
+    Created_On = models.DateTimeField(null=True)
+    District_ID = models.PositiveSmallIntegerField(null=True)
+    District_Name = models.CharField(null=True, max_length=255)
+    HealthBlock_Name = models.CharField(null=True, max_length=255)
+    HealthBlock_ID = models.IntegerField(null=True)
+    Taluka_ID = models.IntegerField(null=True)
+    Taluka_Name = models.CharField(null=True, max_length=255)
+    MDDS_TalukaID = models.IntegerField(null=True)
+    MDDS_DistrictID = models.PositiveSmallIntegerField(null=True)
+    PHC_Name = models.CharField(null=True, max_length=255)
+    PHC_ID = models.IntegerField(null=True)
+    Subcentre_Name = models.CharField(null=True, max_length=255)
+    Village_ID = models.IntegerField(null=True)
+    Mobile_no = models.BigIntegerField(null=True)
+    Religion = models.CharField(null=True, max_length=255)
+    State_Name = models.CharField(null=True, max_length=255)
+    StateID = models.PositiveSmallIntegerField(null=True)
+    Updated_On = models.DateTimeField(null=True)
```

Comment:
Is there a reason to make these camel case? Normally DB columns should be lower case (makes querying easier since you don't have to quote column names)